### PR TITLE
More documentation

### PR DESCRIPTION
**Pull request by armonge**

### CODE REVIEW

File: docs/voxa-platform.rst [hunk 1]
```diff
@@ -4,28 +4,103 @@
...
+  exports.handler = alexaSkill.lambda();
+
+
+
+.. _dialogflow-platform:
+
+DialogFlow
+-------------
+
+The DialogFlow Platform let's you use Voxa with DialogFlow
```
Comment (omenocal) [inline docs/voxa-platform.rst]: let*, no let's

or `allows you to use` ?

File: docs/voxa-platform.rst [hunk 2]
```diff
@@ -4,28 +4,103 @@
...
 
-  .. code-block:: javascript
+.. _botframework-platform:
 
-      exports.handler = app.lambda();
+Botframework
+------------------
 
-.. js:function:: VoxaPlatform.lambdaHTTP()
+The BotFramework Platform let's you use Voxa with Microsoft Botframework
```
Comment (omenocal) [inline docs/voxa-platform.rst]: Same here

File: docs/voxa-platform.rst [hunk 3]
```diff
@@ -4,28 +4,103 @@
...
+  const config = require('./config');
 
-  .. code-block:: javascript
+  const tableName = config.tableName;
+  const storageKey = config.storageKey; // Obtain from Azure Portal
+  const azureTableClient = new AzureTableClient(tableName, storageName, storageKey);
+  const tableStorage = new AzureBotStorage({ gzipData: false }, azureTableClient);
 
-      exports.handler = app.lambdaHTTP();
+  const botframeworkSkill = new BotFrameworkPlatform(app, {
```
Comment (omenocal) [inline docs/voxa-platform.rst]: Isn't it `voxaApp` here ?

File: docs/voxa-event.rst [hunk 4]
```diff
@@ -3,15 +3,35 @@
...
+
+    The current translation function from i18next, initialized to the language of the current request
+
+  .. js:attribute:: renderer
+
+    The renderer object used in the current request
+
+  .. js:attribute:: platform
+
+    A string attribute with the name of platform for the current request, it can be alexa or dialogflow
```
Comment (omenocal) [inline docs/voxa-event.rst]: or microsoft/cortana ?

Should we add support in docs for Cortana in the docs ?

File: docs/alexa-apis.rst [hunk 5]
```diff
@@ -330,103 +324,101 @@ Alexa customers have access to two default lists: Alexa to-do and Alexa shopping
...
 
-  :param listId: List ID.
-  :param itemId: Item ID.
-  :param value: Item name.
-  :param status: item status. Values accepted: 'active | completed'
-  :returns Object: A JSON object with the specific item info.
+    :param listId: List ID.
+    :param itemId: Item ID.
+    :param value: Item name.
+    :param status: item status. Values accepted: 'active | completed'
```
Comment (omenocal) [inline docs/alexa-apis.rst]: Uppercase `Item`

Comment (armonge) [reply docs/alexa-apis.rst]: @omenocal btw, if only those 2 values are accepted, please create a type with those 2 values instead of just saying `string`

**Pull request merged**